README.md

```diff
@@ -11,16 +11,39 @@ Image Template to build custom notebook images for JupyterHub.
 ## How to build the image
 
 ### Locally
+
+#### Prerequisites
+
+* **s2i**: We assume majority of users will be using linux and recommend using the the instructions
+[here](https://github.com/openshift/source-to-image#for-linux) to install s2i if needed.
+
+* **Podman**: Please follow the instructions [here](https://github.com/openshift/source-to-image#for-linux) for your
```

Comment:
You don't really need Podman and Docker both. 
You can use either one of them to run the image it's the users preference.

Reply:
Maybe I don't fully understand how it all works, but it seems like the s2i build relies on docker and leaves the image in your local docker repository. 

Is there a way and should we add documentation for forcing the s2i builder to use podman instead of docker if that is the users' preference?

Reply:
I don't think s2i works with podman yet, what I was trying to say is the user could not use podman and just work with docker and it should work fine

Reply:
Ok, so should we take out the podman requirements and just assume docker will be used?  Except in the running the image locally bit below, we can give alternative directions for using podman there if they want. 

But over all just assume docker as the default?

Reply:
yeah I think using the `docker` cli in instructions, just makes the users' lives easier. 

> we can give alternative directions for using podman there if they want

I like that idea :+1: